domain/secret/state/types.go

```diff
@@ -44,24 +44,26 @@ type secretRef struct {
 }
 
 type secretMetadata struct {
-	ID             string    `db:"secret_id"`
-	Version        int       `db:"version"`
-	Description    string    `db:"description"`
-	AutoPrune      bool      `db:"auto_prune"`
-	RotatePolicyID int       `db:"rotate_policy_id"`
-	CreateTime     time.Time `db:"create_time"`
-	UpdateTime     time.Time `db:"update_time"`
+	ID                     string    `db:"secret_id"`
+	Version                int       `db:"version"`
+	Description            string    `db:"description"`
+	AutoPrune              bool      `db:"auto_prune"`
+	RotatePolicyID         int       `db:"rotate_policy_id"`
+	CreateTime             time.Time `db:"create_time"`
+	UpdateTime             time.Time `db:"update_time"`
+	LatestRevisionChecksum string    `db:"latest_revision_checksum"`
 }
 
 // secretInfo is used because sqlair doesn't seem to like struct embedding.
```

Comment:
struct embedding supported now, can we fix this as a drive by

Reply:
They are a bit different. For example, the secretMetadata.RotatePolicyID != secretInfo.RotatePolicy.